addons/website_sale/views/snippets.xml

```diff
@@ -34,4 +34,68 @@
     </xpath>
 
 </template>
+
+<template id="remove_external_snippets" inherit_id="website.external_snippets">
+    <xpath expr="//t[@t-install='website_sale']" position="replace"/>
+</template>
+
+<template id="website_sale_snippet" inherit_id="website.snippets">
+    <xpath expr="//div[@id='snippet_feature']//t[@t-snippet][last()]" position="after">
+        <t t-snippet="website_sale.s_products_searchbar" t-thumbnail="/website/static/src/img/snippets_thumbs/s_products_searchbar.png"/>
+    </xpath>
+    <xpath expr="//div[@id='snippet_content']//t[@t-snippet][last()]" position="after">
+        <t t-snippet="website_sale.o_wsale_products_searchbar_input" t-thumbnail="/website/static/src/img/snippets_thumbs/s_products_searchbar.png"/>
+    </xpath>
+</template>
+
+<template id="website_sale_snippet_options" inherit_id="website.snippet_options">
+    <xpath expr="." position="inside">
+        <div data-js="ProductsSearchBar"
+            data-selector="form.o_wsale_products_search_bar_input">
+            <a tabindex="-1" href="#" class="dropdown-item" data-no-preview="true" data-open-searchbar-settings=""><i class="fa fa-search"></i>Products Search Bar</a>
+        </div>
+    </xpath>
+    <xpath expr="//div[@data-js='content']" position="attributes">
+        <attribute name="data-selector" add=".o_wsale_products_searchbar_input" separator=","/>
+        <attribute name="data-drop-near" add=".o_wsale_products_searchbar_input" separator=","/>
+    </xpath>
+</template>
+
+<!-- Products Search Bar input-group template -->
+<template id="website_sale_products_search_box" inherit_id="website.website_search_box">
+    <xpath expr="//input[@name='search']" position="attributes">
+        <attribute name="class" add="o_wsale_search_products" separator=" "/>
+        <attribute name="data-display-description">true</attribute>
+        <attribute name="data-display-price">true</attribute>
+        <attribute name="data-display-image">true</attribute>
+    </xpath>
+    <xpath expr="//div[hasclass('input-group')]" position="replace">
+        <form class="o_wsale_products_search_bar_input o_website_sale_search" t-att-action="action if action else '/shop'" method="get">
+            <t>$0</t>
+            <input name="order" type="hidden" class="o_wsale_search_order_by" value="name asc"/>
+            <t t-raw="0"/>
+        </form>
+    </xpath>
+</template>
+
+<!-- Products Search Bar snippet template -->
+<template id="s_products_searchbar" name="Search Bar">
+    <section class="s_wsale_products_searchbar pt24 pb24">
+        <div class="container">
+            <div class="row justify-content-center">
```

Comment:
Cannot use `justify-content-center` on this row, otherwise the column will not be editable properly. Remove it and add `offset-lg-2` on the column.